krun.py

```diff
@@ -277,9 +314,27 @@ def main():
     config = util.read_config(config_file)
     out_file = util.output_name(config_file)
 
+    mail_recipients = config.get("MAIL_TO", [])
+    if type(mail_recipients) is not list:
+        fatal("MAIL_TO config should be a list")
+
+    max_mails = config.get("MAX_MAILS", 5)
+
+    # You should only have mailing for real experiments.
+    # Prevent accidental sending of mail during benchmark implementation
+    if mail_recipients:
```

Comment:
This clause is still redundant. The user knows it's going to send mail N times, because they told it ito. Let's not repeat the sins of some Unix's dmesg ;)